blocks/radio-input/__input/radio-input__input.css

```diff
@@ -0,0 +1,3 @@
+.radio-input__input:checked + .radio-input__label::after{
```

Comment:
@wwily Hey, I apologize for only noticing this now. I think we should move all the code from this file: `radio-input__input.css`, to the styles file of the `radio-input__label.css` element.

Right now, we have the following:

```css
/* radio-input__input.css */
.radio-input__input:checked + .radio-input__label::after{
    opacity: 1;
}

/* radio-input__label.css */
.radio-input__input:hover + .radio-input__label::before{
    border: 2px solid rgba(40, 54, 69);
}

.radio-input__input:focus + .radio-input__label::before{
    border: 2px solid rgba(210, 40, 86);
}
```
Each of these three selectors styles the label's pseudo-elements depending on the input state. Do you see how structurally similar they are? 
`.radio-input__input:checked + .radio-input__label::after`
`.radio-input__input:hover + .radio-input__label::before`
`.radio-input__input:focus + .radio-input__label::before`

So, let's move all the styles from the `radio-input__input.css` file to the `.radio-input__label.css` file. We can then delete the `__input` folder, and don't forget to remove the corresponding import in the `components.css` file.

Reply:
done^^

Reply:
Thank you!